.circleci/config.yml

```
@@ -1,5 +1,36 @@
 version: 2
 jobs:
+  unit-php54-mage18:
+    docker:
+      - image: italoc/php54
+      - image: circleci/mysql:5.7
```

use the ram version

---

I think it's as simple as
```suggestion
      - image: circleci/mysql:5.7-ram
```